clients/src/main/java/org/apache/kafka/common/requests/OffsetFetchRequest.java

```diff
@@ -136,18 +134,10 @@ public OffsetFetchResponse getErrorResponse(int throttleTimeMs, Errors error) {
             }
         }
 
-        switch (versionId) {
-            case 0:
-            case 1:
-            case 2:
-                return new OffsetFetchResponse(error, responsePartitions);
-            case 3:
-            case 4:
-            case 5:
-                return new OffsetFetchResponse(throttleTimeMs, error, responsePartitions);
-            default:
-                throw new IllegalArgumentException(String.format("Version %d is not valid. Valid versions for %s are 0 to %d",
```

Comment:
I see these checks are now gone and can't find where they get moved to (was expecting them in some common place). Unless I missed it, are these included in some future PR?

Reply:
There isn't a need to check for a "maximum message version" here, because we check for that elsewhere.  So the checks don't need to be moved anywhere else, just removed.